save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/dto/GitRepoDto.kt

```diff
@@ -0,0 +1,19 @@
+package org.cqfn.save.preprocessor.dto
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Data class with repository information
+ *
+ * @param url - url of repo
+ * @param username - username to credential
+ * @param password - password to credential
```

Comment:
Add a note or a fixme comment, that we will operate not with raw passwords but with some sort of token, e.g. from github integration